templates/classic-remix-compiler/fly/tsconfig.json

```diff
@@ -1,19 +1,10 @@
 {
-  "include": [
-    "**/*.ts",
-    "**/*.tsx",
-    "**/.server/**/*.ts",
-    "**/.server/**/*.tsx",
-    "**/.client/**/*.ts",
-    "**/.client/**/*.tsx"
```

Comment:
I though there was some reason we needed to include `.server` and `.client` explicitly for tsconfig?

Reply:
Ah or maybe that was only for the new `.server` and `.client` _directories_ since `.server`/`.client` files don't start with  a `.`. So probably git got confused and these lines never existed for the classic compiler.